knowledge-base/actions/aormsby/fork-sync-with-upstream-action/action-security.yml

```diff
@@ -0,0 +1,3 @@
+name: 'Upstream Sync'
```

Comment:
@nasablackberry  I think this will require the contents:write as they mention using the token to push to target repo

Reference: https://github.com/aormsby/Fork-Sync-With-Upstream-action/blob/04be1c21398b7257929087ffb06ddaa08d6bc31e/run/push_updates.sh#L10

https://github.com/aormsby/Fork-Sync-With-Upstream-action/blob/48f332742cb9d99febdd7ec6fb39696ad5d186f4/README.md?plain=1#L44